Update documentdb-resources.md

greg-shelton opened the pull request:
In the table the id property is listed as "System Generated", but as mentioned in the description it can be set by the user and if not provided will be system generated.

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-shawja). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

greg-shelton committed:
Update documentdb-resources.md

In the table the id property is listed as "System Generated", but as mentioned in the description it can be set by the user and if not provided will be system generated.
